CollAction/Frontend/app/project/PieChart.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
```

Comment:
Rewrote the pie chart here so that it no longer contains inline javascript, and is a proper react component.